### DISCUSSION TITLE
Issue with members and roles when running locally/self-hosted

### DISCUSSION THREAD

**Discussion by gkodikara** [AntlerVC/firetable#1362] [Support / Q&A] [answered]
Hi,

See attached.
I'm having this issue when running locally and self-hosted - new members and existing members don't seem to be pulling through, so I can't manage them.

I've used the provided firestore.rules but I'm not sure what's missing here?

Any help would be greatly appreciated.

<img width="924" alt="Screenshot 2023-07-31 at 8 35 12 am" src="https://github.com/rowyio/rowy/assets/735514/f9321764-71b9-4f94-8919-f50e0d963af1">

**Comment by harinij** [comment, marked as answer]
Hi @gkodikara - User management is available in the hosted version of Rowy via rowy.app. The self-hosted version's user management isn't fully functional or scoped within the test.
